client/components/remove-button/index.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'gridicons';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+export class RemoveButton extends React.Component {
+	static propTypes = {
+		onRemove: PropTypes.func.isRequired,
+		translate: PropTypes.func,
```

Comment:
I'd add a comment separating "inherited" props like `translate` from the "proper" one above.

Also, `translate` should be marked `.isRequired` if present.

Reply:
👍